test/e2e/storage/testsuites/testdriver.go

```diff
@@ -141,11 +141,12 @@ type DriverInfo struct {
 	InTreePluginName string
 	FeatureTag       string // FeatureTag for the driver
 
-	MaxFileSize          int64               // Max file size to be tested for this driver
-	SupportedFsType      sets.String         // Map of string for supported fs type
-	SupportedMountOption sets.String         // Map of string for supported mount option
-	RequiredMountOption  sets.String         // Map of string for required mount option (Optional)
-	Capabilities         map[Capability]bool // Map that represents plugin capabilities
+	MaxFileSize          int64                           // Max file size to be tested for this driver
+	SupportedFsType      sets.String                     // Map of string for supported fs type
+	SupportedMountOption sets.String                     // Map of string for supported mount option
+	RequiredMountOption  sets.String                     // Map of string for required mount option (Optional)
+	Capabilities         map[Capability]bool             // Map that represents plugin capabilities
+	RequiredAccessModes  []v1.PersistentVolumeAccessMode // List of access modes required for provisioning
```

Comment:
I think it should be marked as optional and that it defaults to RWO (when #80377 is merged).